sycl/test-e2e/ESIMD/unified_memory_api/block_load_acc_pvc.cpp

```diff
@@ -0,0 +1,42 @@
+//==------- block_load_usm_pvc.cpp - DPC++ ESIMD on-device test ------------==//
```

Comment:
```suggestion
//==------- block_load_acc_pvc.cpp - DPC++ ESIMD on-device test ------------==//
```

Reply:
This and other comments are addressed in https://github.com/intel/llvm/pull/11594